### DISCUSSION TITLE
MT607 disappears after (during?) Deadstart in COS

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#128] [General]
show_tape and show_all (of course) show nothing, but the heading appears.

Shows detection, but no dropping of unit in startup messages when bypassing node start. 
Very obviously deadstarts, the unit disappears after deadstart (during?).

Defaults used entirely.

Interesting... It shows in the channel context display still, probably explaining the heading...

Anyone else?

**Comment by kej715** [comment]
COS uses 607 tape drives, and DtCyber's device driver for this type of
drive currently does not implement a function for displaying its status.

**Comment by KC5VDJ** [comment]
Okay, so load_tape still works, just no status messaging?

As primitive as this is, it was a quite powerful OS.
